pytest_ethereum/linker.py

```diff
@@ -48,7 +56,8 @@ def _deploy(contract_name: str, args: Any, package: Package) -> Tuple[Package, A
             "necessary to populate manifest deployments that have a link reference. If using the "
             "builder tool, use `contract_type(..., runtime_bytecode=True)`."
         )
-    tx_hash = factory.constructor(*args).transact()
+    tx_data = kwargs.get("transaction")
```

Comment:
If I'm reading this correctly, the only thing that `kwargs` is used for is an optional `transaction` key which can be used to specify the transaction parameters.

What about making that explicit and just having an optional `transaction` keyword arg.